master/buildbot/test/unit/test_status_gerrit.py

```diff
@@ -111,37 +93,27 @@ class TestGerritStatusPush(unittest.TestCase):
     }
     THING_URL = 'http://thing.example.com'
 
-    def run_fake_summary_build(self, gsp, buildResults, finalResult,
-                               resultText, expWarning=False):
-        buildpairs = []
-        i = 0
-        for i in xrange(len(buildResults)):
-            buildResult = buildResults[i]
+    def _get_prepared_gsp(self, *args, **kwargs):
```

Comment:
I moved it out of the class since it does not use `self`.  And I oppose this change since it still does not make any use of `self`.